overlord/snapstate/refresh.go

```diff
@@ -21,17 +21,103 @@ package snapstate
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
 	"sort"
 	"strings"
 
 	"github.com/snapcore/snapd/cmd/snaplock"
+	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/sandbox/cgroup"
 	"github.com/snapcore/snapd/snap"
 )
 
-var (
-	pidsCgroupDir = cgroup.ControllerPathV1("pids")
-)
+func securityTagFromCgroupPath(path string) (securityTag string) {
+	_, leaf := filepath.Split(path)
+	if matched, _ := filepath.Match("snap.*.service", leaf); matched {
+		return strings.TrimSuffix(leaf, ".service")
```

Comment:
Yes it would match here but it would not match later on when the return value is compared with the expected glob built based on the name of the snap we are inspecting.

Reply:
I'll add it now.